lite/jupyterlite-pidgy/package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "author": {
+    "email": "deathbeds@users.noreply.github.com",
+    "name": "deathbeds"
+  },
+  "bugs": {
+    "url": "https://github.com/deathbeds/pidgy/issues"
+  },
+  "dependencies": {
+    "@jupyterlite/kernel": "^0.1.0-alpha.18",
+    "@jupyterlite/pyolite-kernel": "0.1.0-alpha.18",
```

Comment:
not depending on this, anymore, actually... need to see if we can get to subclassing... might have webpack issues...

Reply:
actually, do need